src/ast.c

```diff
@@ -208,11 +208,54 @@ static value_t fl_nothrow_julia_global(fl_context_t *fl_ctx, value_t *args, uint
     return b != NULL && jl_atomic_load_relaxed(&b->value) != NULL ? fl_ctx->T : fl_ctx->F;
 }
 
+arraylist_t parsed_method_stack; // for keeping track of which methods are being parsed
```

Comment:
Question for reviewers (@vtjnash and @JeffBezanson): may multiple threads be parsing at the same time? If so, this should probably be thread local or flisp context local.